### PR TITLE
Ports NSV's AGNCA- whatever its called its the RBMK

### PR DESCRIPTION
**Pull request by maxion12345**
- [x] fix the double template spawn issue
- [x] once this is fixed actually test the fucker
- [x] Replace gas supply with tanks so you can sustain a basic setup without atmos engine loop
- [ ] Change numbers to make startup faster (maybe?)
- [ ] Reactor isn't consuming moderator gases. Intended, or not working?
- [ ] Reactor is too difficult to shut down. Increase control rod effectiveness
- [x] look into if porting constricted plasma gas will alleviate this, or give plasma the same effect
- [ ] nsv's guide to their reactor isn't making the stable reaction it says it should. seems to be some fuckery at work here. Find the fuckery.
- [ ] find a different waste gas, nitryl was a good idea but its a powergaming gas (miasma??? fart reactor?????)

# Document the changes in your pull request

Remove this text and explain what the purpose of your PR is.

Mention if you have tested your changes. If you changed a map, make sure you used the mapmerge tool.
If this is an Issue Correction, you can type "Fixes Issue #6" to link the PR to the corresponding Issue number #6.

Prefix the PR title with [admin] if it involves something admin related. 
Prefix the PR title with [s] if you are fixing an exploit, so that it is not announced on the Yogstation Discord and the server.

Remember: something that is self-evident to you might not be to others. Explain your rationale fully, even if you feel it goes without saying.

# Wiki Documentation

Remove this text and write all information regarding your changes that should be known and documented through the Yogstation Wiki. 
Important documentation information includes, but is not limited to: any numerical values that have been changed, any images that have to be updated, names of specific pages that will be impacted by your changes. 

# Changelog

Edit the changelog below to reflect the changes made by this PR, even if the changes are minor - required for every PR.
If you add a name after the ':cl', that name will be used in the changelog. Leave it empty to use your GitHub name.

:cl:  
rscadd: Added new things  
rscdel: Removed old things  
bugfix: fixed a few things  
wip: This is work in progress  
tweak: tweaked a few things  
soundadd: added a new sound thingy  
sounddel: removed an old sound thingy  
imageadd: added some icons and images  
imagedel: deleted some icons and images  
spellcheck: fixed a few typos  
experimental: This is experimental  
/:cl:

### CODE REVIEW

**Comment by stwordcitizen** [comment]
wtf is a rbmk

File: code/__DEFINES/sound.dm [hunk 1]
```diff
@@ -9,11 +9,12 @@
 #define CHANNEL_BUZZ 1017
 #define CHANNEL_BICYCLE 1016
 #define CHANNEL_VOICE_ANNOUNCE 1015
+#define CHANNEL_REACTOR_ALERT 2014
```
Comment (adamsong) [inline code/__DEFINES/sound.dm]:
```suggestion
#define CHANNEL_REACTOR_ALERT 1014
```

**Comment by maxion12345** [comment]
got work this will not be worked on for a bit

**Comment by ilikesaying** [comment]
> wtf is a rbmk

Soviet Nuclear Reactor
Chernobyl

**Pull request closed without merging**